apps/site/lib/site_web/views/event_view.ex

```diff
@@ -130,6 +130,11 @@ defmodule SiteWeb.EventView do
     event.started_status === :ended
   end
 
+  @spec is_started?(Event.t() | Teaser.t()) :: boolean
```

Comment:
Perhaps a bit nit-picky (feel free to ignore) but would it make sense to name this **has**_started?